test/ng/locationSpec.js

```diff
@@ -1601,6 +1601,25 @@ describe('$location', function() {
     });
 
 
+    it('should rewrite links when the specified rewriteLinks attr is detected', function() {
+      configureTestLink({
+        linkHref: 'link?a#b',
+        attrs: 'force-rewrite',
+        html5Mode: {enabled: true, rewriteLinks:false},
+        supportHist: true
+      });
+      initService({html5Mode:{enabled: true, rewriteLinks:'forceRewrite'},supportHistory:true});
```

Comment:
It wasn't the tests, there is a configuration normalizer that forces configurations to be boolean, that I just updated.